apps/docs/src/principles/technical-guidelines.md

```diff
@@ -36,86 +23,76 @@ Our technical vision describes the goals that we as the development team of **on
+1. 📜 All boolean properties default to `false` (falsy)
 1. 📜 We prevent complex conditions due to big union modelValue types.
    - We either offer specialized components (e.g. separating a number input from a text input)
    - Alternatively, we can make use of [generic components](https://vuejs.org/api/sfc-script-setup.html#generics)
-1. 📜 Components are shipped as native Vue components, not as Web Components
-1. 🔮 We provide translations for standard texts inside components, e.g. "OK", "Cancel" etc. and allow to override them.
+1. 📜 We provide translations for standard texts inside components, e.g. "OK", "Cancel" etc. and allow to override them.
    - Community contributions to our translations are most welcome!
-1. 🔮 We support common navigation patterns, e.g. keyboard shortcuts
-1. 🔮 The implemented components reflect the capabilities of Figma
-   - Exception: We can't limit what will be passed into a slot
-   - Exception: We use number and other property types that can not be declared in Figma
-1. 🔮 We use a shared domain language between UX and DEV
-   - e.g. for component names, variable names, ...
-1. 🔮 All components can be used standalone in a project without affecting styles of the whole application (so they can be used together with other component libraries). Therefore, we provide two CSS files:
-   - Component styles: Required. Defines the styles of the components, no side effects on globals
-   - Global styles: Optional. Applies global styles to the whole application (e.g. color / background color on `body`, `<a>` etc.) <br>
-     Alternatively, a layout component (e.g. OnyxSpace) might be provided for this.
+1. 📜 We support common navigation patterns, e.g. keyboard shortcuts
+1. 📜 We strife to keep the parity between Figma and the implementation as high as possible.
+   - We use a shared domain language between UX and DEV.
+1. 📜 All components can be used standalone in a project without affecting styles of the whole application (so they can be used together with other component libraries). Therefore, we provide two CSS files:
+   - Component styles: Required. Defines the styles of the components, no side effects on globals.
+   - Global styles: Optional. Applies global styles to the whole application (e.g. color / background color on `body`, `<a>` etc.)
 
 ### Naming conventions
 
 1. 📜 All (public) components are prefixed with `Onyx`, e.g. `OnyxButton`
 2. 📜 We use [v-model](https://vuejs.org/guide/components/v-model.html) for two-way-data-binding wherever possible
 3. 📜 Component variants are named after what they are, not where they are used (e.g. a link variant "fullWidth" instead of "sideBar")
-4. 📜 Event names won't get an `onyx` prefix
-5. 📜 Boolean properties are prefixed with `is-` or `has-`. Exceptions are made to align with default html properties, e.g. `required`
-6. 📜 All CSS class names are prefixed with `onyx-`
-7. 📜 All CSS variables are prefixed with `--onyx-`
+4. 📜 All CSS class names are prefixed with `onyx-`
```

Comment:
Should we move the first bullet point here too? So that all "are prefixed with" bullet points are grouped?